README.md

```diff
@@ -111,7 +105,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://github.com/akinmolaomotlani/Hello-project.git)
```

Comment:
- [ ] To make your documentation even better, I have a suggestion regarding the `Live Demo Link`. Since your project doesn't have a live link yet, you may want to consider removing that section from your documentation. This will help to avoid any confusion or frustration for users who might be expecting a live link that isn't available. 💯